Script/AtomicEditor/ui/frames/ProjectFrame.ts

```diff
@@ -382,6 +391,18 @@ class ProjectFrame extends ScriptWidget {
 
     }
 
+    // Shows referenced file in projectframe
+    handleInspectorProjectReferenceHighlight(path: string): void {
+        this.assetReferencePath = path;
+        var db = ToolCore.getAssetDatabase();
+        var asset = db.getAssetByPath(this.resourceFolder.getPath() + "/" + path);
+
```

Comment:
Cool, I always get excited by new asset database and toolcore usage :)  These will also eventually work purely from the CLI :+1: